tests/backend/core/Core/Unit/Logic/PermissionsTest.php

```diff
@@ -129,6 +130,7 @@ protected function getPermissionsInstance(bool $ownsProcedure, bool $inviteOrgaF
         $validator = self::$container->get(ValidatorInterface::class);
         $procedureRepository = $this->getProcedureRepositoryMock();
         $permissionsClass = $this->getPermissionsClass();
+        $accessControlService = self::$container->get(AccessControlService::class);
```

Comment:
question: this is added to potentially be used in the future for permissions in other tests?

Reply:
No, this is added because the it was added to the constructor of `Permissions` and therefore needed to be passed here as otherwise all permission tests broke